docs/source/installation.rst

```diff
@@ -47,10 +47,10 @@ the user can install selectively.
     - `PySpin <https://www.flir.com/products/spinnaker-sdk/>`_ (non-PyPi)
     - `thorlabs_tsi_sdk <https://www.thorlabs.com/software_pages/ViewSoftwarePage.cfm?Code=ThorCam>`_ (non-PyPi)
     - `VimbaPython <https://github.com/alliedvision/VimbaPython>`_ (non-PyPi)
-    - `Xenics SDK <https://www.xenics.com/software/>`_ (non-PyPi)
```

Comment:
I removed it because Xenics is not a python package, and this list otherwise consists only of python packages. The xenics file throws an appropriate error.